Add courses metadata

jgagne opened the pull request:
## What this PR does

- Adds missing courses metadata and assets

### Test

#### Meta

- https://deploy-preview-758--thegymcms.netlify.app/courses/full/gym-101/meta/
- https://deploy-preview-758--thegymcms.netlify.app/courses/full/gym-105/meta/
- https://deploy-preview-758--thegymcms.netlify.app/courses/gym-shorts/gym-002/meta/
- https://deploy-preview-758--thegymcms.netlify.app/courses/gym-shorts/gym-003/meta/

#### Announcement

- https://deploy-preview-758--thegymcms.netlify.app/courses/full/gym-101/announcement/
- https://deploy-preview-758--thegymcms.netlify.app/courses/full/gym-105/announcement/
- https://deploy-preview-758--thegymcms.netlify.app/courses/gym-shorts/gym-002/announcement/
- https://deploy-preview-758--thegymcms.netlify.app/courses/gym-shorts/gym-003/announcement/

jgagne committed:
Add GYM-105 metadata

- Add course data
- Add OG image
- Add course meta
- Add announcement

jgagne committed:
Update list formatting

- Update indent, use 2 spaces

jgagne committed:
Update GYM-105 metadata

- Add missing value

jgagne committed:
Add GYM-003 metadata

- Add course data
- Add OG image
- Add course meta
- Add announcement

rediris commented:
@jgagne we should add one for GYM-101 as well: https://github.com/gymnasium/studio-GYM-101

rediris commented:
@jgagne we should add one for GYM-101 as well: https://github.com/gymnasium/studio-GYM-101

jgagne commented:
@rediris [Totally!](https://github.com/gymnasium/teamwork/projects/6) (Coming soon.) Thanks for setting up the Studio repo!

rediris commented:
> @rediris [Totally!](https://github.com/gymnasium/teamwork/projects/6) (Coming soon.) Thanks for setting up the Studio repo!

No problem! I'm starting to work on the studio repo for 003 as well. Coming up next!

jgagne committed:
Add GYM-101 metadata

- Add course data
- Add OG image
- Add course meta
- Add announcement

jgagne committed:
Add GYM-002 metadata

- Add course data
- Add OG image
- Add course meta
- Add bio and headshot
- Add announcement

jgagne committed:
Update announcement language

- Update GYM-003 "access" to "view"
- Update GYM-101 retired message

jgagne committed:
Update GYM-002 course data

- Remove invalid trailing UTF-8 octet (gremlin)

rediris commented on _data/courses/GYM-002.yml:
```diff
@@ -0,0 +1,24 @@
+course_type: short
+live: false
+retired_message:
+  1: "<strong>This course has been retired.</strong> Due to outdated content, we dont offer a badge for this course  but you can still view the lessons."
```
add an apostrophe for `don't`?

rediris replied:
Or a fancy apostrophe? `don’t`

jgagne replied:
Thanks for the extra 👀 on this!

rediris replied:
No problem! If you're still taking requests, there appears to be an extra space between `course  but`…

jgagne committed:
Update GYM-002 course data

- Add removed punctuation
- Update file encoding

jgagne committed:
Add GYM-002 OG image

- Add missing OG image, ref: dae3365b

jgagne committed:
Update GYM-002 course data

- Add removed ellipsis

jgagne committed:
Update GYM-002 announcement

- Update course number
